packages/editor/src/components/parts/program/editor/EditorPart.tsx

```diff
@@ -0,0 +1,77 @@
+import { EditorData, Label, Script, Text, Widget } from '@axonivy/inscription-protocol';
+import { PartProps, usePartDirty, usePartState } from '../../../editors';
+import { useEditorContext, useMeta, useValidations } from '../../../../context';
+import { useEditorData } from './useEditorData';
+import { Input, MessageText, ScriptInput } from '../../../../components/widgets';
+
+export function useEditorPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = useEditorData();
+  const compareData = (data: EditorData) => [data.userConfig, data.link];

```

Comment:
what is this link for?

Reply:
honestly i don't know exactly either. i have now removed the link.